src/main/java/com/kat/bachaat/configuration/HibernateConfiguration.java

```diff
@@ -1,17 +1,13 @@
 package com.kat.bachaat.configuration;
 
-import org.hibernate.SessionFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.PropertySource;
 import org.springframework.core.env.Environment;
```

Comment:
i dont think this should be your task to remove unused import.

Reply:
done